ibft/commit.go

```diff
@@ -99,7 +102,21 @@ func (i *Instance) uponCommitMsg() pipeline.Pipeline {
 			i.Logger.Info("commit iBFT instance",
 				zap.String("Lambda", hex.EncodeToString(i.State.Lambda)), zap.Uint64("round", i.State.Round),
 				zap.Int("got_votes", len(sigs)))
-
+			var decided *proto.SignedMessage
```

Comment:
@nivBlox I think we should have a function to aggregate separate sigs, please refactor

Reply:
done